Gemfile

```diff
@@ -104,7 +104,6 @@ gem 'holidays', '~> 2.2.0', '< 3.0.0'
 gem 'iso_country_codes', '~> 0.7.0'
 gem 'mahoro', '~> 0.4'
 gem 'newrelic_rpm'
-gem 'net-http-local', '~> 0.1.0', :platforms => [:ruby_19]
```

Comment:
This commit should also remove the code that used the gem:

* https://github.com/mysociety/alaveteli/blob/develop/lib/quiet_opener.rb#L3-L5
* https://github.com/mysociety/alaveteli/blob/develop/lib/quiet_opener.rb#L46-L50